File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Which height? ")

    if height <= 8 and height >= 1:
        break

for i in range(height):
    for j in range(height - i - 1):
        print(" ", end = "")
    for l in range(2):
        for k in range(i + 1):
            print("#", end = "")
        if l == 0:
            print("  ", end = "")
    print("")

File: world/mario.c
#include <cs50.h>
#include <stdio.h>

void print_line(int width, int height);

int main(void)
{
    int height;
    do
    {
        height = get_int("What is the height? \n");
    }
    while (height < 1);

    for (int j = 0; j < height; j++)
    {
        print_line(j, height);
        printf("\n");
    }
}

void print_line(int width, int height)
{
    for (int r = 0; r < height - width; r++)
    {
        printf(" ");
    }

    for (int i = 0; i < width + 1; i++)
    {
        printf("#");
    }

    printf(" ");

    for (int e = 0; e < width + 1; e++){
        printf("#");
    }
}
